package.json

```diff
@@ -22,7 +22,7 @@
     "connected-react-router": "^6.1.0",
     "css-loader": "1.0.0",
     "decentraland-commons": "^5.1.0",
-    "decentraland-dapps": "^4.0.0",
+    "decentraland-dapps": "4.0.1-rc",
```

Comment:
is this the final version?

Reply:
Nope, but there is a PR open on dapps: https://github.com/decentraland/decentraland-dapps/pull/41/files

Basically fixes the default choice so it can be the first item of the provided locales instead of always "en". Feel free to review/merge/anything if it's blocking.